### PR TITLE
Makefile: support LIBC and DESTDIR options.

### PR DESCRIPTION
**Pull request by jialez0**
Fixes:#39

### CODE REVIEW

**Comment by jialez0** [comment]
Hi, @liangzhou121, this PR will resolve your issue :)

File: Makefile [hunk 1]
```diff
@@ -11,19 +10,20 @@ TARGET_DIR := ./target
 BIN_NAME := attestation-agent
 
 DEBUG ?=
-MUSL ?=
+LIBC ?=
```
Comment (dubek) [inline Makefile:13]: various files in .github/workflows use `MUSL=1` in some places. These should be updated to the new syntax.

Comment (jialez0) [reply Makefile:13]: Done.

**Review by fitzthum — APPROVED** [review]
LGTM

**Pull request merged**